pkg/health/server.go

```diff
@@ -0,0 +1,92 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package health
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/dapr/dapr/pkg/logger"
+)
+
+// Server is the interface for the healthz server
+type Server interface {
+	Run(context.Context, int) error
+	Ready()
```

Comment:
nit: naming SetReadyState() and UnsetReadyState() or keep the other one as NotReady() if want to continue the current naming..

Reply:
I changed to `NotReady`, its a better fit.